frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -117,13 +119,23 @@ export const DetachedManagePatients = ({
   const [archivePerson, setArchivePerson] = useState<Patient | null>(null);
   const navigate = useNavigate();
 
+  const [redirect, setRedirect] = useState<
+    string | { pathname: string; search: string; state?: any } | undefined
+  >(undefined);
   const [queryString, debounced, setDebounced] = useDebounce<string | null>(
     null,
     {
       debounceTime: SEARCH_DEBOUNCE_TIME,
     }
   );
 
+  const { data: queueData } = useQuery(queueQuery, {
+    fetchPolicy: "no-cache",
+    variables: {
+      facilityId: activeFacilityId,
```

Comment:
One concern I have here is that this could theoretically surface one of our existing bugs, where a patient is added to the queue at facility A while already being in-queue at facility B. It's probably not a big deal here since that can already happen from the queue, but it may be worth prioritizing #3103  after this change goes in.

Reply:
Another out-of-scope improvement I would recommend here is to make a new graphQL query to "fetchLimitedQueue" or similar. The existing queue query fetches all details of a test order (device, ordering provider, ask on entry questions, etc) - it's a somewhat expensive query. All you really want here is the list of patients on the queue, none of the other details about the test orders. Again this is out of scope for this change, but something to consider if these changes prove expensive.

Reply:
@emmastephenson I think the more limited query is a good idea - I went ahead and wrote a [ticket](https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/3586) so we don't forget, but it would be worth a bit of investigation re: performance to justify the effort.